source/_patterns/01-molecules/components/suggestions.twig

```diff
@@ -0,0 +1,7 @@
+<ul class="o-suggestions">
```

Comment:
Add the `u-border--bottom` class to this line.

Reply:
Also, to make the link lighter, add `u-theme--color--base` and `u-theme--link-hover--base` to this `ul`.

Reply:
Please use this updated list of styles on the `ul` class: `u-theme--color--light u-link-hover--white u-border--bottom`